docs/source/commands/supergraphs.mdx

```diff
@@ -135,20 +135,22 @@ The ELv2-licensed `supergraph` plugin (built from [this source](https://github.c
 
 The `supergraph compose` command supports both Federation 2 and Federation 1 composition. 
 
-> ⚠️ Note that if you use Federation 1 composition and _any_ of your subgraphs uses a Federation 2 schema, composition will fail!
+> ⚠️ Warning: Future versions of `rover supergraph compose` will fail without specifying an exact federation version. Set this configuration to prevent breaking changes in future Rover versions.
 
 Federation 1 and Federation 2 use different composition algorithms, which are implemented in different libraries:
 
 * **Federation 1:** [`@apollo/federation`](https://www.npmjs.com/package/@apollo/federation)
 * **Federation 2:** [`@apollo/composition`](https://www.npmjs.com/package/@apollo/composition)
 
+> ⚠️ Note that if you use Federation 1 composition and _any_ of your subgraphs uses a Federation 2 schema, composition will fail!
+
 Whenever you run `rover supergraph compose`, Rover automatically downloads the version of the library for your selected federation version. 
 
 > ⚠️ **The `federation_version` you target must not exceed the highest version supported by your router!** Make sure to update your router _before_ incrementing your `federation_version`. For details, see this [support table](/router/federation-version-support).
 
```

Comment:
I think the order of info in the "Setting a composition version" should be as follows:

1. Whenever you run `rover supergraph compose`, Rover automatically downloads the version of the library for your selected federation version. --> include how you select a federation version
2. Warnings and notes about specifying federation version.

This would be my suggestion specifically:
(Unfortunately cannot include the suggestion in the comment since this covers deleted lines)

-----

Whenever you run `rover supergraph compose`, Rover automatically downloads the federation version you specified either in your [supergraph.yaml](#yaml-configuration-file) or [via stdin](#supergraph-compose).

The command supports both Federation 1 and Federation 2 composition.  Federation 1 and Federation 2 use different composition algorithms, which are implemented in different libraries:

* **Federation 1:** [`@apollo/federation`](https://www.npmjs.com/package/@apollo/federation)
* **Federation 2:** [`@apollo/composition`](https://www.npmjs.com/package/@apollo/composition)

<blockquote>

⚠️ Important

- The federation version you specify must not exceed the highest version supported by your router!** Make sure to update your router _before_ incrementing your `federation_version`. For details, see this [support table](/router/federation-version-support).
- If you specify a Federation 1 composition and _any_ of your subgraphs uses a Federation 2 schema, composition fails.
- Future versions of `rover supergraph compose` will fail if you don't specify an exact federation version. Include a federation version in your configuration to prevent breaking changes in future Rover versions.

</blockquote>

---

(I'm using an old-fashioned blockquote instead of <Warning> to align with the styling on the rest of the page.)